lib/worker-pipeline.js

```diff
@@ -173,8 +174,8 @@ class AssetComputeWorkerPipeline {
         // normalize source and rendition
         this.normalizeInputOuput(input, output);
         // TODO: type check to make sure input/output are objects so we don't throw here if they are not?
-        debug(`Input for refinePlan: ${JSON.stringify(input, null, 1)}`);
-        debug(`Output for refinePlan: ${JSON.stringify(output, null, 1)}`);
+        AssetComputeLogUtils.log(input,'Input for refinePlan:');
+        AssetComputeLogUtils.log(output,'Output for refinePlan:');
```

Comment:
could you share what this output looks like? (I remember it was tricky to get right when I originally looked into this: https://github.com/adobe/asset-compute-sdk/pull/131/files#diff-78b14646706f0830f6321b56729ae22332f1afbf2d2782808adf7b6142deda8c)

ideally, for debugging purposes, I'd like to keep the whole nested json logged, just with the urls redacted:
```
Output for refinePlan: {
 "features": {
  "autoTags": {
   "image": false,
   "text": {
    "maximumWords": 3,
    "minimumConfidence": 0.7
   },
   "video": {
    "minimumActionConfidence": 0.7,
    "minimumObjectConfidence": 0.7
   }
  },
  "colorTags": false,
  "featureVector": false
 },
 "fmt": "machine-metadata-json",
 "pipeline": true,
 "target": {
  "maxPartSize": 104857600,
  "minPartSize": 10485760,
  "urls": [ ...REDACTED...]
 },
 "userData": {
  "assetComputeClient": {
   "index": 0,
   "length": 1
  },
  "path": "75d46622-de3e-478a-a3d2-30ca81f29582/machine-metadata-json/bulldog-5071407_319x179.png/rendition0.machine-metadata-json"
 },
 "type": "machine-metadata-json"
}
```
[old logs](https://adobeiort.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22runtime_prod_ext%22%20activation_id%3D72174fbeb2b54751974fbeb2b5575101%20%7C%20sort%20_time%20%7C%20fields%20%2B%20_time%2Clog_message%20%7C%20fields%20-%20_raw%20%7C%20outputtext&display.page.search.mode=smart&dispatch.sample_ratio=1&earliest=-7d%40h&latest=now&display.page.search.showFields=0&display.events.type=table&display.page.search.tab=statistics&display.general.type=statistics&sid=1626971142.341352)

Reply:
I didn't get a chance to run e2e tests. I verified the logs in build/test.logs by changing the params.json for `pipeline-png-webp` test

```
{
	"pipeline": true,
	"fmt": "webp",
        "accessToken": "SOME_accessToken"
	"errorReason": "RenditionFormatUnsupported"
}
```

output:
```
2021-07-22T13:51:12.238234200Z Output for refinePlan: : { pipeline: true,
2021-07-22T13:51:12.238274100Z   fmt: 'webp',
2021-07-22T13:51:12.238294100Z   accessToken: '[...REDACTED...]',
2021-07-22T13:51:12.238308200Z   type: 'image/webp' }
```

Reply:
@jdelbick @pheenomenon Normally the filenames should be redacted too.